skills/relay-review/scripts/review-runner/assurance.js

```
@@ -1,63 +1,181 @@
+const LANE_DEMOTION_CAP = 2;
+
 function buildAssuranceIssue(title, body) {
   return {
     title,
     body,
     file: "policy.review_assurance",
     line: 1,
     category: "quality",
+    confidence: "high",
```

Biome (2.5.1)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)